server/src/main/scala/drt/server/feeds/lhr/forecast/LhrForecastArrivals.scala

```diff
@@ -0,0 +1,94 @@
+package drt.server.feeds.lhr.forecast
+
+import drt.server.feeds.lhr.forecast.LhrForecastArrivals.getClass
+import drt.shared.{Arrival, SDateLike}
+import org.joda.time.DateTimeZone
+import org.slf4j.{Logger, LoggerFactory}
+import services.SDate
+
+import scala.util.{Failure, Success, Try}
+
+object LhrForecastArrivals {
+  val log: Logger = LoggerFactory.getLogger(getClass)
+
+  def apply(lines: Seq[String]): Seq[Arrival] = {
+    lines
+      .map(line => LhrForecastArrival(line))
+      .filterNot {
+        case Success(_) => false
```

Comment:
wouldn't it make more sense to return true on success, and false on failure? Call me old fashioned.

Reply:
good catch - i was originally trying to report the failed cases, hence the double negative